cfgov/v1/__init__.py

```diff
@@ -186,10 +186,10 @@ def related_metadata_tags(context, page):
     # From an ancestor, get the form ids then use the first id since the
     # filterable list on the page will probably have the first id on the page.
     id, filter_page = get_filter_data(page)
-    for tag in page.specific.tags.names():
-        tag_link = {'text': tag, 'url': ''}
+    for tag in page.specific.tags.all():
+        tag_link = {'text': tag.name, 'url': ''}
         if id is not None and filter_page is not None:
-            param = '?filter' + str(id) + '_topics=' + tag
+            param = '?filter' + str(id) + '_topics=' + tag.slug
```

Comment:
@vickumar1981 [slugs](https://github.com/alex/django-taggit/blob/master/taggit/models.py#L25') are [already url-safe](https://docs.djangoproject.com/en/1.10/ref/models/fields/#slugfield).

That said, this might be cleaner with Python 3-compatible string formatting, something like:

```
param = '?filter{id}_topics={slug}'.format(id=str(id), slug=tag.slug).
```

But that's not a necessary change. I would very much like to avoid introducing any more old-style string formatting though.